ast/node.go

```diff
@@ -721,9 +727,11 @@ func (self *NullLiteral) Idx1() file.Idx        { return file.Idx(int(self.Idx)
 func (self *NumberLiteral) Idx1() file.Idx      { return file.Idx(int(self.Idx) + len(self.Literal)) }
 func (self *ObjectLiteral) Idx1() file.Idx      { return self.RightBrace + 1 }
 func (self *ObjectPattern) Idx1() file.Idx      { return self.RightBrace + 1 }
+func (self *ParameterList) Idx1() file.Idx      { return self.Closing }
```

Comment:
Should be `self.Closing + 1`, and in expression.go:1179 it should be `Closing: id.Idx1() - 1`